### PR TITLE
[RELEASE] v1.0.3

### PR DESCRIPTION
**Pull request by jinnoflife**

### CODE REVIEW

File: src/Components/CancelService/CancelService.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            $chargeId,
+            $amountGross,
+            '',
+            '',
+            $amountNet,
+            $amountVat
+        );
+    }
+
+    protected function getOrderTransaction(string $orderTransactionId, Context $context): ?OrderTransactionEntity
```
Comment (sixer1182) [inline src/Components/CancelService/CancelService.php:77]: Some describing comments would be nice.
Please add for next release.

File: src/Components/CancelService/CancelService.php [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        $taxRates = [];
+
+        /** @var CalculatedTax $calculatedTax */
+        foreach ($transaction->getAmount()->getCalculatedTaxes() as $calculatedTax) {
+            $taxRates[] = $calculatedTax->getTaxRate();
+        }
+
+        $clearedTaxRate = array_sum($taxRates) / count($taxRates);
+
+        $roundedAmountGross = (int) round($amountGross * (10 ** $decimalPrecision));
```
Comment (sixer1182) [inline src/Components/CancelService/CancelService.php:58]: Comment in next release please.

**Review by sixer1182 — CHANGES_REQUESTED** [review]
3 places version please

File: composer.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "unzerdev/shopware6",
     "description": "Unzer payment integration for Shopware 6",
-    "version": "1.0.2",
+    "version": "1.0.3.0",
```
[the remark this answers is not in the document]
Comment (jinnoflife) [reply composer.json]: done

**Review by sixer1182 — APPROVED** [review]
LGTM

**Pull request merged**